.evergreen/.evg.yml

```diff
@@ -277,7 +286,7 @@ functions:
           elif [ -f venv/Scripts/activate ]; then
             . venv/Scripts/activate
           fi
-          mongo-orchestration stop
+          mongo-orchestration stop || true
```

Comment:
The embedded tests don't run mongo-orchestration so don't fail the test if it doesn't exist.